src/components/Account-panel/Change-password/Change-password.tsx

```diff
@@ -15,19 +17,29 @@ type FormData = {
   new_password_confirm: string
 }
 
-const stInput = {
+export const stInput = {
```

Comment:
remove this styles, we already have most of them for `Input`, use only you need insisde the component `sx` prop, not as a variable

Reply:
We don't have anything new of this type, I think so far